tasks/create_target_vm/01_create_target_hosted_engine_vm.yml

```diff
@@ -119,7 +119,7 @@
     register: storage_domain_details
   - debug: var=storage_domain_details
   - name: Add HE disks
-    ovirt_disk:
+    ovirt_disk_27:
```

Comment:
Can you please rename it as ovirt_disk_28 and add a comment asking to remove it once we will require ansible 2.8?